### PR TITLE
Added Azure pipelines example

### PR DESCRIPTION
**Pull request by e-minguez**
I cannot create a public project in my Azure account so you can see if it works (IDK why) but it worked :)

image

### CODE REVIEW

**Comment by e-minguez** [comment]
@airadier @manuelbcd PTAL Thanks

**Review by airadier — CHANGES_REQUESTED** [review]
Thanks very much, this is _lovely_. Two_ minor things:

- Can we move the examples inside the new-scan-engine folder? We are trying to reorganize the repository so we keep legacy and new engine examples under different folders
- In the index.md file state clearly that the example is for the new engine.

**Comment by e-minguez** [comment]
> Thanks very much, this is _lovely_
🤣 
>. Two_ minor things:
> 
> * Can we move the examples inside the new-scan-engine folder? We are trying to reorganize the repository so we keep legacy and new engine examples under different folders
Addressed, thanks.
> * In the index.md file state clearly that the example is for the new engine.
Addressed, thanks.

PTAL and thanks for the feedback!

File: docs/index.md [hunk 1]
```diff
@@ -130,6 +130,8 @@ The `no_proxy` variable can be used to define a list of hosts that don't use the
 
 In this [repository](https://github.com/sysdiglabs/secure-inline-scan-examples/) you can find the following examples in alphabetical order:
 
+* [AWS Codebuild](https://github.com/sysdiglabs/secure-inline-scan-examples/tree/main/new-scan-engine/aws-codebuild)
+* [Azure Pipelines](https://github.com/sysdiglabs/secure-inline-scan-examples/tree/main/azure-pipelines)
```
Comment (airadier) [inline docs/index.md:134]: Please include "New scan engine" here (see Gitlab example below)

**Review by airadier — APPROVED** [review]
Sorry for the confusion and noise, let's align on this with @manuelbcd because I see we have a:

/new-scan-engine/aws-codebuild

with an example for aws-codebuild. Then for gitlab, we have:

/gitlab/old-scan-engine
/gitlab/new-scan-engine

which follows this patterm. Which one we should choose and stick with? Make the /new-scan-engine at the root level and put examples inside, or make the split at the integration level?

I am ok with the current approach (just see my minimal change proposal) and we make the refactor later if needed.

File: docs/index.md (hunk 1 above)
Comment (airadier) [inline docs/index.md:134]:
```suggestion
* [Azure Pipelines (New scan engine)](https://github.com/sysdiglabs/secure-inline-scan-examples/tree/main/azure-pipelines)
```

**Comment by e-minguez** [comment]
> Sorry for the confusion and noise, let's align on this with @manuelbcd because I see we have a:
> 
> /new-scan-engine/aws-codebuild
> 
> with an example for aws-codebuild. Then for gitlab, we have:
> 
> /gitlab/old-scan-engine /gitlab/new-scan-engine
> 
> which follows this patterm. Which one we should choose and stick with? Make the /new-scan-engine at the root level and put examples inside, or make the split at the integration level?
> 
> I am ok with the current approach (just see my minimal change proposal) and we make the refactor later if needed.

My 2 cents on this is to have them splitted by the provider:

```
GitHub/
|-- new-scanner-engine
`-- old-scanner-engine
GitLab/
|-- new-scanner-engine
`-- old-scanner-engine
```

**Comment by manuelbcd** [comment]
Thanks @e-minguez , but just a minor change in the folder names to align with the current ones: instead of "scanning" we are using "scan"

GitHub/
|-- new-scan-engine
`-- old-scan-engine
GitLab/
|-- new-scan-engine
`-- old-scan-engine

thanks!

**Review by manuelbcd — APPROVED** [review]

**Pull request merged**